.github/workflows/ci.yaml

```diff
@@ -446,6 +446,77 @@ jobs:
         run: |
           poetry run pytest tests/ -vvv -n 2 ${PYTEST_OPTS}  ${{ matrix.PYTEST_OPTS }}
 
+  min-btc-support:
+    name: Test BTC version 25.0 ${{ matrix.name }}
```

Comment:
Could test `25.2` here instead? That is the latest release on the `25.x` branch.

Also, looking at https://github.com/ElementsProject/lightning?tab=readme-ov-file#getting-started, I guess the minimum required verison is now 25.x rather than 22.0?

Reply:
@fanquake Thanks for your review. I've updated the README accordingly. However, maintaining the minimum supported BTC version at 25.0 to accommodate versions released till two years ago. Version 25.0 was released in May 2023, while version 25.2 was released in April 2024.